store/mongo/datastore_mongo.go

```diff
@@ -535,6 +535,71 @@ func (db *DataStoreMongo) UpdateDevicesGroup(
 	}, nil
 }
 
+func (db *DataStoreMongo) GetFiltersAttributes(ctx context.Context) ([]model.FilterAttribute, error) {
+	database := db.client.Database(mstore.DbFromContext(ctx, DbName))
+	collDevs := database.Collection(DbDevicesColl)
+
+	const DbCount = "count"
+
+	cur, err := collDevs.Aggregate(ctx, []bson.M{
+		{
+			"$project": bson.M{
+				"attributes": bson.M{
+					"$objectToArray": "$" + DbDevAttributes,
+				},
+			},
+		},
+		{
+			"$unwind": "$" + DbDevAttributes,
+		},
+		{
+			"$project": bson.M{
+				DbDevAttributesName:  "$" + DbDevAttributes + ".v." + DbDevAttributesName,
+				DbDevAttributesScope: "$" + DbDevAttributes + ".v." + DbDevAttributesScope,
+			},
+		},
```

Comment:
Should we add a `$skip` and limit stage `$limit` here?
The `$group` stage is allowed to allocate up to 100MiB of RAM before it returns an error, it might be safer to add pagination to the API and limit the usage?

Reply:
@alfrunes good question, I believe pagination is not going to be very efficient, though, as we have to run the aggregation over and over. I'm more in favor of a fixed $limit (e.g. `500`), as the main usage of this end-point is to provide autocompletion in the UI. What do you think?

Reply:
Hmm.. I guess a fixed limit makes more sense when you put it like that. It would be pretty awkward if the endpoint suddenly stopped working above a certain device limit, I guess we can come back and tweak the exact limit at a later stage if we feel like it but 500 sounds plenty.